blog/_posts/2018-04-24-open-source-charts-2017.md

```diff
@@ -0,0 +1,48 @@
+
+Read on to hear how the Kubernetes Community has addressed these scaling challenges to reach the top of the charts in GitHub’s 2017 Octoverse report.
+
+## Most-Discussed on GitHub
+
+The top two most-discussed repos of 2017 are both based on Kubernetes:
+
+![Most Discussed](/images/blog-logging/2018-04-24-open-source-charts-2017/most-discussed.png)
+
+Of all the open source repositories on GitHub, none received more issue comments than [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/). [OpenShift](http://openshift.com/), a [CNCF certified distribution of Kubernetes](https://blog.openshift.com/kubernetes-and-openshift-community-standards-and-certifications/
+https://www.cncf.io/announcement/2017/11/13/cloud-native-computing-foundation-launches-certified-kubernetes-program-32-conformant-distributions-platforms/), took second place.  
+
+Open discussion with ample time for community feedback and review helps build shared infrastructure and establish new standards for cloud native computing.
+
+## Most Reviewed on GitHub
+
+Successfully scaling an open source effort’s communications often leads to better coordination and higher-quality feature delivery. The Kubernetes project’s [Special Interest Group (SIG)](https://github.com/kubernetes/community/blob/master/sig-list.md) structure has helped it become GitHub’s second most reviewed project:
+
+![Most Reviewed](/images/blog-logging/2018-04-24-open-source-charts-2017/most-reviews.png)
+
+Using SIGs to segment and standardize mechanisms for community participation helps channel more frequent reviews from better-qualified community members.
+
+When managed efficiently, active community discussions indicate more than just a highly contentious codebase, or a project with an extensive list of unmet needs.
+
+Scaling a project’s capacity to handle issues and community interactions helps to expand the conversation.  Meanwhile, large communities come with more diverse use cases and a larger array of support problems to manage. The Kubernetes [SIG organization structure](https://github.com/kubernetes/community#sigs) helps to address the challenges of complex communication at scale.
+
+SIG meetings provide focused opportunities for users, maintainers, and specialists from various disciplines to collaborate together in support of this community effort.  These investments in organizing help create an environment where it’s easier to prioritize architecture discussion and planning over commit velocity; enabling the project to sustain this kind of scale.
+
+## Join the party!
+
+You may already be using solutions that are successfully managed and scaled on Kubernetes. For example, GitHub.com (which hosts Kubernetes’ upstream source code) [now runs on Kubernetes](https://githubengineering.com/kubernetes-at-github/) https://githubengineering.com/kubernetes-at-github/) as well!
```

Comment:
Use commas instead of parentheses:
"For example, GitHub.com, which hosts Kubernetes’ upstream source code, [now runs on Kubernetes](https://githubengineering.com/kubernetes-at-github/) https://githubengineering.com/kubernetes-at-github/) as well!"